### PR TITLE
Update installing-mojaloop.md

### PR DESCRIPTION
**Pull request by kjw000**
updated install doc

### CODE REVIEW

File: docs/getting-started/installation/installing-mojaloop.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+
+1. Manual Deployent. The Mojaloop [Deployment Guide](https://docs.mojaloop.io/legacy/deployment-guide/) is intended for those familiar with [Kubernetes](https://kubernetes.io) and [Helm](https://helm.sh).
+
+2. IaC (Infrustructure as Code).  Comprehensive Mojaloop deployment aimed at giving users a starting point for production. IaC is highly automated (terraform, ansible).  To learn more about an [IaC Deployment](https://infitx.com/deploying-mojaloop-using-iac/) review the blog.   IaC currently supports the following configurations: Amazon Web Services (AWS): [IaC AWS Github](https://github.com/mojaloop/iac-aws-platform), Azure and is working on On-Premise.
+
+3. [Mini-Loop](https://github.com/tdaly61/mini-loop) install utilities for Mojaloop and offer an easy and efficient way to get started. The [Mini-Loop](https://github.com/tdaly61/mini-loop) scripts enable you to deploy Mojaloop in the cloud or on your laptop / server with just a couple of commands. You can then easily run the [Mojaloop Testing Toolkit](https://github.com/mojaloop/ml-testing-toolkit#mojaloop-testing-toolkit) to interact and test your deployment.
```
Comment (tdaly61) [inline docs/getting-started/installation/installing-mojaloop.md:10]: can we please keep mini-loop as mini-loop (no M) in this case.

File: docs/getting-started/installation/installing-mojaloop.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+4. [Azure Marketplace](https://github.com/mojaloop/documentation-artifacts/blob/master/presentations/pi_21_march_2023/presentations/Mojaloop%20Azure%20Deployment.pdf)  This is an Azure AKS native deployment and aims to give users a starting point for POC or pilot testing.  It is a simple deployment, with highly automated ARM templates and deploys to managed Kubernetes for easier managment. Run the [Mojaloop Testing Toolkit](https://github.com/mojaloop/ml-testing-toolkit#mojaloop-testing-toolkit) to interact and test your deployment.
+
+5. Azure StackHub.  Similar end-state to Azure (native) but on Stack HUB (data residence).  It deploys to managed Kubernetes for easier management.
```
Comment (tdaly61) [inline docs/getting-started/installation/installing-mojaloop.md:14]: So for Stack Hub there is a document produced by Sybrin and Microsoft but this is not do-able without that doc which i am not sure is available.
So perhaps leave this option out for the moment.

File: docs/getting-started/installation/installing-mojaloop.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+
+3. [Mini-Loop](https://github.com/tdaly61/mini-loop) install utilities for Mojaloop and offer an easy and efficient way to get started. The [Mini-Loop](https://github.com/tdaly61/mini-loop) scripts enable you to deploy Mojaloop in the cloud or on your laptop / server with just a couple of commands. You can then easily run the [Mojaloop Testing Toolkit](https://github.com/mojaloop/ml-testing-toolkit#mojaloop-testing-toolkit) to interact and test your deployment.
+
+4. [Azure Marketplace](https://github.com/mojaloop/documentation-artifacts/blob/master/presentations/pi_21_march_2023/presentations/Mojaloop%20Azure%20Deployment.pdf)  This is an Azure AKS native deployment and aims to give users a starting point for POC or pilot testing.  It is a simple deployment, with highly automated ARM templates and deploys to managed Kubernetes for easier managment. Run the [Mojaloop Testing Toolkit](https://github.com/mojaloop/ml-testing-toolkit#mojaloop-testing-toolkit) to interact and test your deployment.
```
Comment (tdaly61) [inline docs/getting-started/installation/installing-mojaloop.md:12]: would change ARM to MIcrosoft ARM

File: docs/getting-started/installation/installing-mojaloop.md (hunk 1 above)
Comment (kjw000) [reply docs/getting-started/installation/installing-mojaloop.md:10]: yes

File: docs/getting-started/installation/installing-mojaloop.md (hunk 2 above)
Comment (kjw000) [reply docs/getting-started/installation/installing-mojaloop.md:14]: will remove

**Review by tdaly61 — APPROVED** [review]
looks good , thanks for doing this Kim

**Pull request merged**